### DISCUSSION TITLE
Notice subtype 10 visual model example

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#484] [Q&A] [answered]
Could you show us a correct and complete example of a visual model (not generated by the Notice Editor Demo application) of a notice subtype 10?

**Comment by rouschr** [comment, marked as answer]
There is no such thing as a "complete visual model", each application can have a different visual model and handle things differently (present fields in a different order, have extra groupings...).
The editor demo generates one possible visual model from the editor demo form. The form itself is built from the SDK `notice-types/10.json` metadata.
The common point is that you need to be able to go from a form with data to the structure of **fields** and **nodes** (fields.json, conceptual model) in order to go to the **XML** and have the data put at the right place and in the right order (to the physical model). Unless other applications open their source code the editor demo will be the closest you will get to see on how to build a notice editor using SDK. Note that our work on the editor is not done and the SDK is improving with each version, also thanks to feedback via the seminar and discussions / reports made here.
